src/BRSRC13/CORE/FW/datafile.c

```diff
@@ -308,15 +308,18 @@ int TextReadLine(br_datafile* df, char** ident, char** data) {
     LOG_TRACE9("(%p, %p, %p)", df, ident, data);
 
     while (1) {
+        int a;
         cp = BrScratchString();
-        int a = BrFileGetLine(cp, 256, df->h);
+        a = BrFileGetLine(cp, 256, df->h);
+        (void)a;
```

Comment:
what is this doing?

Reply:
I silences a warning about setting a variable, but not using it.
It's probably better to just remove the `a = ` thing.